plugins/node/opentelemetry-instrumentation-ioredis/package.json

```
@@ -69,7 +69,7 @@
   "dependencies": {
     "@opentelemetry/instrumentation": "^0.51.0",
     "@opentelemetry/redis-common": "^0.36.2",
-    "@opentelemetry/semantic-conventions": "^1.0.0"
+    "@opentelemetry/semantic-conventions": "^1.23.0"
```

nit: This should be `^1.22.0` right?

```suggestion
    "@opentelemetry/semantic-conventions": "^1.22.0"
```

---

I just though if we are making an update, might as well update to the latest, so I did this on purpose. The message on readme says about 1.22+ to accommodate when this would happen.